Subsetting.Rmd

```diff
@@ -108,23 +108,24 @@ There are five things that you can use to subset a vector:
 
     ```{r}
     x[c(TRUE, TRUE, FALSE, FALSE)]
-    x[x > 3]
-    ```
-
+    x[x > 3]                            # Note that here 3 represents the value, not the index
+    ```				
+	
     If the logical vector is shorter than the vector being subsetted, it 
     will be _recycled_ to be the same length.
-
-    ```{r}
-    x[c(TRUE, FALSE)]
-    # Equivalent to
-    x[c(TRUE, FALSE, TRUE, FALSE)]
-    ```
+	
+	
+		     ```{r}
+       x[c(TRUE, FALSE)]
+       # Equivalent to
+       x[c(TRUE, FALSE, TRUE, FALSE)]
+            ```
 
     A missing value in the index always yields a missing value in the output:
 
-    ```{r}
+    ```{r}																										
     x[c(TRUE, TRUE, NA, FALSE)]
-    ```
+        ```
```

Comment:
Sorry for this formatting non-sense: I don't use rmarkdown (I use org-babel) and some of my edits messed up the rmarkdown formatting in emacs polymode, that I then tried to fix.